### ISSUE TITLE
Charts' Pie3D series labels can't be hidden with Display="None"

### ISSUE THREAD

**Issue by fabriciomurta** [extnet/Ext.NET#1379]
This is a bug both in Ext.NET and Sencha ExtJS.

Sencha thread: [Pie chart: label display 'none' does not work](https://www.sencha.com/forum/showthread.php?288550)
Ext.NET Thread: [Hiding labels on polar 3D pie chart](http://forums.ext.net/showthread.php?61418)
#### Ext.NET part of the bug (#1380)

The assumed default for `label.display` is `None`, but that's not the case, at least with the _Pie3D_ component's label. In fact, when defined, a label's display property should be anything but `None`.

So, when we specify a label and set `Display="None"`, Ext.NET does not output the change. The enum for this property should have a `Default` option which would be assumed as default so when it is specified in Ext.NET it does not get output to the JavaScript resulting code.
#### ExtJS part of the bug

ExtJS ignores the `label.display: none` setting at all. A simple, pragmatic override that prevents the label from being drawn by all means would be:

``` javascript
Ext.define('Ext.chart.series.Pie3D', {
    override: 'Ext.chart.series.Pie3D',
    updateLabelData: function () {
        return true;
    }
});
```

The problem has already been fixed by ExtJS, and will be released with ExtJS 6.0.3, already available on sencha fiddle for preview version 6.2.0.981.

Sencha fiddle address: [Pie3D Chart: Unable to hide the label](https://fiddle.sencha.com/#fiddle/1gj3)

_Remarks:_
- It is set up to ExtJS 6.0.2 so it will reproduce the issue and highlight the workaround
- You can switch the fiddle version to ExtJS 6.2.0.981 or newer to see the first chart fixed.

**Comment by fabriciomurta** [comment]
Issue #1380 was logged to avoid closing this ExtJS-related issue when merged without fixing Ext.NET part of the issue!

**Comment by fabriciomurta** [comment]
**Update:** Sencha issue _EXTJS-14177_ allegedly fixed in ExtJS 6.0.3 release.
